publish.js

```diff
@@ -473,6 +473,21 @@ var copyRecursiveSync = function(src, dest) {
     }
 };
 
+function getVersions () {
+    var topLevelOutDir = path.normalize(env.opts.destination);
```

Comment:
Good idea. Never tried to access git tags via Node.js. But I guess there is a package that could make that work. Guess  it would be straight forward, If tags are named exactly like version numbers from the package.json.

But in case of `v1.0.0` we would need to rid of the prefix.

So we could have an option like:
```
"versionSwitcher": {
  "versionsFromGitTags": true,
  "gitVersionPrefix": "v" 
}
```